lib/auth/webauthn/config.go

```diff
@@ -0,0 +1,33 @@
+package webauthn
+
+import (
+	"github.com/duo-labs/webauthn/protocol"
+
+	wan "github.com/duo-labs/webauthn/webauthn"
+)
+
+const (
+	defaultDisplayName = "Teleport"
+	defaultIcon        = ""
+)
+
+// Config represents the Webauthn configuration.
+// TODO(codingllama): Replace with types.Webauthn once it's merged.
+type Config struct {
+	RPID                                        string
+	AttestationAllowedCAs, AttestationDeniedCAs []string
+}
+
+func newWebAuthn(cfg *Config, rpID, origin string) (*wan.WebAuthn, error) {
+	var attestation protocol.ConveyancePreference
```

Comment:
What's the default `ConveyancePreference`? I would be explicit about it.

Reply:
If we don't say anything it defaults to "none". Changed code to assign it explicitly.

Reply:
Can you do so without the else?
```suggestion
	attestation := protocol.PreferNoAttestation
	if len(cfg.AttestationAllowedCAs) > 0 && len(cfg.AttestationDeniedCAs) > 0 {
	     attestation = protocol.PreferDirectAttestation
	}
	
```

Reply:
Also, do you want `||` instead of `&&` in the conditional? It is possible to specify just allowed or just denied, right?

Reply:
Well spotted, done.

(I'll have tests for this when we get to the attestation impl.)